projects/plugins/jetpack/modules/wordads/js/wordads-ccpa.js

```diff
@@ -312,7 +312,8 @@
 					if ( 200 === this.status ) {
 						// Got a geo response. Parse out the region data.
 						var data = JSON.parse( this.response );
-						var ccpaApplies = data.region && 'california' === data.region.toLowerCase();
+						var region      = data.region ? data.region.toLowerCase() : '';
+						var ccpaApplies = ['california', 'colorado', 'connecticut', 'utah', 'virginia'].includes( region );
```

Comment:
Let's change from `.includes( region );` to `.indexOf( region ) > -1;` to match the WPCOM implementation.